components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/handlers/TokenResponseTypeHandler.java

```diff
@@ -565,4 +562,16 @@ private boolean isNotRenewAccessTokenPerRequest() {
         }
         return !isRenew;
     }
+
+    private static AccessTokenDO getAccessTokenCheckingContext(String accessToken,
+                                                               OAuthAuthzReqMessageContext msgCtx) throws IdentityOAuth2Exception {
```

Comment:
shall we conform to the 120 character length policy?

Reply:
Ok,I'll check